.github/workflows/release.yml

```diff
@@ -60,6 +60,12 @@ jobs:
       with:
         node-version: 14.x
 
+    # The AVM requires openssl dev files to build now, and apparently the rust openssl code can auto-acquire it through vcpkg, so set that up
+    - name: Set up vcpkg
+      run: |
+        git clone https://github.com/microsoft/vcpkg
```

Comment:
should we pin to a specific tag?

Reply:
So this is a Microsoft repository for a Microsoft compiler and Microsoft operating system. [I am also following the official install instructions](https://docs.microsoft.com/en-us/cpp/build/install-vcpkg?view=msvc-160&tabs=windows) so I think this is fine?